bigearthnet/configs/config.yaml

```diff
@@ -65,5 +65,6 @@ hydra:
 
 defaults:
   - _self_
+  - override hydra/sweeper: orion
```

Comment:
This line should have gone at the end of file.

Reply:
Fixed.